shared.csv

```diff
@@ -29,3 +29,6 @@ ArchUnit,Tools,Trial,TRUE
 Mailgun,Services,Trial,TRUE
 Grafana,Infrastructure Components,Trial,TRUE
 Prometheus,Infrastructure Components,Trial,TRUE
+Toxiproxy,Tools,Trial,TRUE
+Firefox Relay,Tools,Trial,TRUE
```

Comment:
Where do we use this one?

Reply:
I use it for our manual tests to generate temporary e-mail addresses to test sign-up. I once mentioned it in https://camunda.slack.com/archives/C01B2SF0DKM/p1607586302001700 and had added a link to it in our manual QA sheet.

Reply:
I also used some other services for that purpose and there are a lot of them like temp-mail, guerrillamail and others. Because this is  something that is only used individually, I would not add it to the technology radar. Otherwise we could also add all online tools that we have used like JWT online validation, JWK online generation etc...

Reply:
Yup, that makes sense :+1: I removed Firefox Relay